src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String[] FIGURES = {"Square", "RightTriangle", "Rectangle",
+            "IsoscelesTrapezoid","Circle"};
+    private static final String DEFAULT_COLOR = "White";
+    private static final int DEFATUL_LENGTH = 10;
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+
+    FigureSupplier(Random random, ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
```

Comment:
you need to choose between constructor initialization or initialization with declaration and be consistent 
I'll explain, 
in the **line 11** you are creating the object of the Random, creating the object here 
In the **line 10** you are only declaring the variable and initializing it after, **in the 14 line**
So, you have two types here, you should choose one  and make it with both random and colorSupplier objects

For now, in 13 line random parameter is redundant